Gemfile

```diff
@@ -32,7 +32,7 @@ group :coverage, optional: true do
 end
 
 group :rubocop, optional: true do
-  gem 'rubocop', '~> 0.88.0'
+  gem 'rubocop', '~> 0.89.0'
```

Comment:
I was going to ask why not `0.93.1`, but in fact `1.0.0` has just been released today.

Reply:
I find Rubocop updating is quite a lot of work for not much gain, so I didn't want to jump more versions than required to get CI going again. I'll bump to 1.0 though, as it should now be a lot more stable

Reply:
Fingers crossed!  Of course I'd be happy to leave it at 0.89 and factor out the move to 1.0 for another day if it's not straightforward.

Reply:
Turns out to have been super easy & it caught a duplicate `require` 😀